rustls/src/client/client_conn.rs

```diff
@@ -122,7 +122,8 @@ pub trait ResolvesClientCert: Send + Sync {
 /// (the rustls-native-certs crate is often used for this) may take on the order of a few hundred
 /// milliseconds.
 ///
-/// These must be created via the [`ClientConfig::builder()`] function.
+/// These must be created via the [`ClientConfig::builder()`] or [`ClientConfig::builder_with_provider()`]
```

Comment:
I think this doesn't need to be part of this commit?

Reply:
hoisted these drive-by docs changes into their own commit